<launch>
    <group ns="/science/panorama" >
        <node pkg="filesend" type="file_recv.py" name="fsend_recv" output="screen">
            <param name="save_to" value="$(env HOME)/.urc/pano_work_base" />
        </node>

        <node pkg="rover_panorama" name="panorama_orches" type="panorama_orchestrator.py" output="screen">
            <param name="image_save_dir" value="$(env HOME)/.urc/pano_work_base" />
        </node>
    </group>
</launch>